8/ansible-8.build

```diff
@@ -99,7 +99,6 @@ purestorage.flasharray: >=1.18.0,<2.0.0
 purestorage.flashblade: >=1.11.0,<2.0.0
 purestorage.fusion: >=1.4.0,<2.0.0
 sensu.sensu_go: >=1.13.0,<2.0.0
-servicenow.servicenow: >=1.0.0,<2.0.0
```

Comment:
The collection MUST NOT be removed from Ansible 8. This is a breaking change and thus not acceptable since Ansible sticks to semantic versioning.

Reply:
I don't think this is the only place we have to change in this case... let me have a look and do another commit.

Reply:
@felixfontein I think it should be OK now.